core/toolbox.js

```diff
@@ -533,7 +533,9 @@ Blockly.Toolbox.TreeNode = function(toolbox, html, opt_config, opt_domHelper) {
   goog.ui.tree.TreeNode.call(this, html, opt_config, opt_domHelper);
```

Comment:
Yes, I meant why is not declared as `Blockly.Toolbox.prototype.TreeNode` since it takes an instance of a `Blockly.Toolbox` anyway.
Can the `this` argument from `goog.ui.tree.TreeNode.call` be from the declaration `Blockly.Toolbox.prototype.TreeNode`? Javascript is not really one of my primary languages, so this is an area I'm not too familiar with, I assumed in both cases you would end up with the same type of `this`, just "attached" to different things.